src/Stratis.Bitcoin/P2P/Peer/NetworkPeerServer.cs

```diff
@@ -112,23 +138,23 @@ public NodeServer(Network network, ProtocolVersion version = ProtocolVersion.PRO
             this.messageProducer.AddMessageListener(listener);
             this.OwnResource(listener);
 
-            this.ConnectedNodes = new NodesCollection();
-            this.ConnectedNodes.Added += Nodes_NodeAdded;
-            this.ConnectedNodes.Removed += Nodes_NodeRemoved;
-            this.ConnectedNodes.MessageProducer.AddMessageListener(listener);
+            this.ConnectedNetworkPeers = new NetworkPeerCollection();
+            this.ConnectedNetworkPeers.Added += Peers_PeerAdded;
+            this.ConnectedNetworkPeers.Removed += Peers_PeerRemoved;
+            this.ConnectedNetworkPeers.MessageProducer.AddMessageListener(listener);
 
             this.AllMessages = new MessageProducer<IncomingMessage>();
-            this.trace = new TraceCorrelation(NodeServerTrace.Trace, "Node server listening on " + this.LocalEndpoint);
+            this.trace = new TraceCorrelation(NodeServerTrace.Trace, "Network peer server listening on " + this.LocalEndpoint);
```

Comment:
Did we not get rid of `TraceCorrelation`?

Reply:
Ah its temporary ok fine

Reply:
PR too big already, getting rid of this did not make it in this phase